plugins/rainlab/user/models/User.php

```diff
@@ -160,8 +160,7 @@ public function getAvatarThumb($size = 25, $options = null)
     {
         if (is_string($options)) {
             $options = ['default' => $options];
-        }
```

Comment:
Davajte si pozor aby vam automaticky neformatovalo subory, ktore nemaju nic spolocne s tym o com je ten pull request, lebo to tu teraz zavadzia a je to cluttter